website/src/components/PreviewModeBar.js

```diff
@@ -30,9 +34,13 @@ export default function PreviewModeBar({ postId, position = 'top' }) {
       <div className="container">
         <div className="row">
           <div className="col text-center">
-            You are viewing this site in Preview Mode.{' '}
+            <span className={styles.icon}>
+              <BsInfoCircle />
+            </span>
+            You are viewing this site in Preview Mode
             {postId && (
               <>
+                &nbsp;&nbsp;|&nbsp;&nbsp;
```

Comment:
Can this be a `<span style={{ display: 'inline-block', paddingLeft: '5px' }}>`?

Reply:
we're not using inline styles anywhere else, I could add it to SCSS i suppose